c3po/api/controller/feed_controller.py

```diff
@@ -0,0 +1,42 @@
+from flask import abort, request
+from flask_restx import Resource, Namespace, reqparse
+from flask_restx.inputs import datetime_from_iso8601
+from ..service.feed_service import FeedService
+from ..dto import FeedDto
+
+api = FeedDto.api
+songObject = FeedDto.songObject
+
+parser = reqparse.RequestParser()
+parser.add_argument('limit', type=int)
+parser.add_argument('genre', type=str)
+parser.add_argument('from', type=datetime_from_iso8601)
+parser.add_argument('to', type=datetime_from_iso8601)
+
+
+@api.route('/popular')
+class FeedPopular(Resource):
+    """ User Login Resource """
+    @api.doc('Popular songs (most liked)')
+    @api.marshal_list_with(songObject)
+    @api.expect(parser)
+    def get(self): 
+        args = parser.parse_args()
+        resp = FeedService.get_from_to(args['from'], args['to'])
+        
+        if resp[1] != 200:
+            abort(403, resp[0])
+        else:
+            return resp
```

Comment:
Can we make this more verbose? These methods are returning a tuple, so we can assign it to one. So it becomes,
```py
resp, code = FeedService.get_from_to(args['from'], args['to'])

if code != 200:
    abort(403, resp)
else:
    return resp
```

This way, I don't have to look at the method to understand what is happening since the variable names are self-explanatory.

Reply:
Yep, ok. I'll change this.